src/WebJobs.Script/Description/DotNet/CSharp/CSharpCompilationService.cs

```diff
@@ -18,34 +18,23 @@ namespace Microsoft.Azure.WebJobs.Script.Description
     [CLSCompliant(false)]
     public class CSharpCompilationService : ICompilationService
     {
-        private readonly IFunctionMetadataResolver _metadataResolver;
+        private static readonly string[] FileTypes = { ".csx", ".cs" };
         private static readonly Encoding UTF8WithNoBOM = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
         private static readonly Lazy<InteractiveAssemblyLoader> AssemblyLoader
           = new Lazy<InteractiveAssemblyLoader>(() => new InteractiveAssemblyLoader(), LazyThreadSafetyMode.ExecutionAndPublication);
 
         private readonly OptimizationLevel _optimizationLevel;
+        private readonly IFunctionMetadataResolver _metadataResolver;
 
         public CSharpCompilationService(IFunctionMetadataResolver metadataResolver, OptimizationLevel optimizationLevel)
         {
             _metadataResolver = metadataResolver;
             _optimizationLevel = optimizationLevel;
         }
 
-        public string Language
```

Comment:
Just some cleanup and small optimizations here since I was working in this area.